website/docs/r/vpn_server_configuration_policy_group.html.markdown

```diff
@@ -0,0 +1,93 @@
+---
+subcategory: "Network"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_vpn_server_configuration_policy_group"
+description: |-
+  Manages a VPN Server Configuration Policy Group.
+---
+
+# azurerm_vpn_server_configuration_policy_group
+
+Manages a VPN Server Configuration Policy Group.
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "example-resources"
+  location = "West Europe"
+}
+
+resource "azurerm_vpn_server_configuration" "example" {
+  name                     = "example-VPNSC"
+  resource_group_name      = azurerm_resource_group.example.name
+  location                 = azurerm_resource_group.example.location
+  vpn_authentication_types = ["Radius"]
+
+  radius {
+    server {
+      address = "10.105.1.1"
+      secret  = "vindicators-the-return-of-worldender"
+      score   = 15
+    }
+  }
+}
+
+resource "azurerm_vpn_server_configuration_policy_group" "example" {
+  name                        = "example-VPNSCPG"
+  vpn_server_configuration_id = azurerm_vpn_server_configuration.example.id
+
+  policy_member {
+    name            = "policy1"
+    attribute_type  = "RadiusAzureGroupId"
+    attribute_value = "6ad1bd08"
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The Name which should be used for this VPN Server Configuration Policy Group. Changing this forces a new resource to be created.
+
+* `vpn_server_configuration_id` - (Required) The ID of the VPN Server Configuration that the VPN Server Configuration Policy Group belongs to. Changing this forces a new resource to be created.
+
+* `policy_member` - (Required) One or more `policy_member` blocks as documented below.
+
+* `is_default` - (Optional) Is this a default VPN Server Configuration Policy Group? Defaults to `false`. Changing this forces a new resource to be created.
+
+* `priority` - (Optional) The priority of this VPN Server Configuration Policy Group. Defaults to `0`.
+
+---
+
+A `policy_member` block supports the following:
```

Comment:
member? what does member mean here and would this be better
```suggestion
A `policy` block supports the following:
```

Reply:
Updated